build-release.sh

```diff
@@ -1,87 +1,102 @@
-#!/usr/bin/env bash
```

Comment:
Not certain if that wasn't because it broke in some environments :thinking: ... what is the upside of not using it ?

Reply:
The standard Bash location on most Linux systems is `/bin/bash`. It's safer to not rely on `env` if you don't need it, since it avoids the uncertainty of which binary will be used. Otherwise it would run with any `bash` executable in `PATH`, which could not only be incompatible versions but also malicious interpreters. This is unlikely of course, but since this script should only be run in a controlled CI environment, we don't need the cross-platform benefit of using `env`.